### ISSUE TITLE
Integrate redux dev tools

### ISSUE THREAD

**Issue by dev-tim** [WorldBrain/webmarks-client#1]

**Comment by grady-lad** [comment]
@dev-tim  The project supports this [plugin](https://github.com/zalmoxisus/redux-devtools-extension#redux-devtools-extension) should we just use this for the redux dev tools ?

**Comment by dev-tim** [comment]
Yes, why not. We need to check what is about store configuration because of server side rendering setup.
